regimes/au/au.go

```diff
@@ -0,0 +1,68 @@
+// Package au provides the Australian tax regime.
+package au
+
+import (
+	"github.com/invopop/gobl/bill"
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/currency"
+	"github.com/invopop/gobl/i18n"
+	"github.com/invopop/gobl/tax"
+)
+
+func init() {
+	tax.RegisterRegimeDef(New())
+}
+
+const (
+	TaxCategoryWET cbc.Code = "WET"
```

Comment:
Constants also need comments, either for the comment block or the specific example. You can run the linter locally (`golanglint-ci`), or see the Github suggestions.